telegram/_bot.py

```diff
@@ -1171,6 +1186,8 @@ async def send_document(
             api_kwargs (:obj:`dict`, optional): Arbitrary keyword arguments to be passed to the
                 Telegram API.
 
+        .. seealso:: :attr:`telegram.Message.reply_document`,  :attr:`telegram.Chat.send_document`
```

Comment:
```suggestion
        .. seealso:: :attr:`telegram.Message.reply_document`, :attr:`telegram.Chat.send_document`
```
found another space :]